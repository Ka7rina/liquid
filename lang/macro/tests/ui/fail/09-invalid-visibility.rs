use liquid_lang as liquid;

#[liquid::contract(version = "0.1.0")]
mod noop {
    #[liquid(storage)]
    struct Noop {}

    #[liquid(methods)]
    impl Noop {
        #[liquid(constructor)]
        fn init(&mut self) {}

        pub(crate) fn noop(&self) {}
    }
}

fn main() {}
