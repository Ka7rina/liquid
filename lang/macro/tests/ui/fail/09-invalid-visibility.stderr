error: crate-level visibility or visibility level restricted to some path is not supported for liquid functions
  --> $DIR/09-invalid-visibility.rs:13:9
   |
13 |         pub(crate) fn noop(&self) {}
   |         ^^^^^^^^^^
